README.md

```diff
@@ -179,29 +192,94 @@ try {
 }
 ```
 
-#### Retrieving a file
+##### Example: Storing multiple files
+
+```js
+/**
+ * Example usage to store multiple files.
+ */
+
+// The hashes of the desired files.
+const hashes = [{
+  algorithm: 'SHA-256',
+  value: '8f434346648f6b96df89dda901c5176b10a6d83961dd3c1ac88b59b2dc327aa4',
+}, {
+  algorithm: 'SHA-256',
+  value: 'ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad',
+}];
+
+// First, check if the files are already in COS.
+try {
+  // This triggers a permission prompt. For example:
+  // example.com wants to check if your browser already has files the site needs,
+  // possibly saved from another site. If found, it will use the files without
+  // changing them.
+  // [Allow this time] [Allow on every visit] [Don't allow]
+  const handles = await navigator.crossOriginStorage.requestFileHandles(hashes);
+  // The files exist in COS.
+  for (const handle of handles) {
+    const fileBlob = await handle.getFile();
+    // Do something with the blob.
+    console.log('Retrieved', fileBlob);
+  }
+  return;
+} catch (err) {
+  // If the files weren't in COS, load them from the network and store them in
+  // COS.
+  if (err.name === 'NotFoundError') {
+    try {
+      // Load the files from the network.
+      const fileBlobs = await loadFilesFromNetwork();
+      const handles = await navigator.crossOriginStorage.requestFileHandles(
+        hashes,
+        {
+          create: true,
+        },
+      );
+      handles.forEach((handle, i) => {
+        // The resulting `FileSystemFileHandle` can only be used for writing.
+        // Trying to call `handle.getFile()` would fail with a `NotAllowed`
+        // `DOMException`.
+        const writableStream = await handle.createWritable();
+        await writableStream.write(fileBlobs[i]);
```

Comment:
The sequences of hashes and fileBlobs must match, this is probably not obvious.

Reply:
Yes, it's a bit unhandy, but also not the worst. `Promise.all([])` has the same matching challenges frequently. I think it's okay for now.